packages/react-i18n/README.md

```diff
@@ -141,6 +141,7 @@ The provided `i18n` object exposes many useful methods for internationalizing yo
   - `DateStyle.Short`: e.g., `Dec 20, 2012`
   - `DateStyle.Humanize`: Adheres to [Polaris guidelines for dates with times](https://polaris.shopify.com/content/grammar-and-mechanics#section-dates-numbers-and-measurements), e.g., `Just now`, `3 minutes ago`, `4 hours ago`, `10:35 am`, `Yesterday at 10:35 am`, `Friday at 10:35 am`, or `Dec 20 at 10:35 am`, or `Dec 20, 2012`
   - `DateStyle.Time`: e.g., `11:00 AM`
+- `formatDateTime()`: formats a date and time according to the Polaris guidlines for [dates with times](https://polaris.shopify.com/foundations/content/grammar-and-mechanics#date).
```

Comment:
Remember to update this to be another enum description like `DateStyle.ShortDateTime` and also include the key that needs to be translated to go with it.

Reply:
👍 Sounds great! This was a relic from back in the day when I had it defined as a public method instead of a DateStyle 😉 .